src/components/display/components/InputComponent.tsx

```diff
@@ -1,16 +1,34 @@
 import React, { useState } from 'react'
 import { InputArgs } from 'display-api'
 import { Button, TextInput } from 'react-native-paper'
-import { StyleSheet, Text, View } from 'react-native'
+import { Platform, StyleSheet, Text, View } from 'react-native'
 import { globalStyles } from '../../../utils/styles'
 import { blockMethod, setMethodVariable } from '../method'
+import DateTimePicker from '@react-native-community/datetimepicker'
+import { MaterialCommunityIcons } from '@expo/vector-icons'
+import colors from '../../../utils/colors'
+import moment from 'moment'
 
-export const InputComponent = ({ initial_value, name, label, mask, confirm_cancel }: InputArgs) => {
+export const InputComponent = ({
```

Comment:
Looks good! Just needs the `frequency` type now. I'll extract that to another ticket and close this one, because it deserve's it's own, looks like.